### DISCUSSION TITLE
Whitelabel Error Page

### DISCUSSION THREAD

**Discussion by JHub-Wei** [apache/skywalking#8192] [Q&A]
We visit the ui homepage and report an error Whitelabel Error Page, using version 8.7.0, using containerized deployment。
What is the reason for this? And what is the solution?
![ui](https://user-images.githubusercontent.com/48902832/143671368-2633dd65-3da5-436e-8505-71c3ffdd0635.PNG)

**Comment by JHub-Wei** [comment]
I encountered this problem while upgrading from 8.4.0 to 8.7.0. If the cache is not cleared, it can be accessed. Once the cache is cleared, an error Whitelabel Error Page is reported.

**Comment by JHub-Wei** [comment]
The backend address is specified through the environment variable SW_OAP_ADDRESS.

**Comment by wu-sheng** [comment]
I can't see what is going on at your side, sorry. We never tested upgrading OAP server with using old version's database. The structures of index/table are changed from version to version.

**Reply by JHub-Wei** [reply]
We did not use old version's database, and cleared the data and templates in es before the upgrade.

**Reply by wu-sheng** [reply]
Then nothing is wrong. We have many users using all these versions, including our demo env and showcase(https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/).
All are running well.
So, sorry, no clue.
